{"3781ce1fd0a25de5feef610905320f32":{"statistic":{"failed":0,"broken":1,"skipped":0,"passed":0,"unknown":0,"total":1},"items":[{"uid":"52df137ff87f04bb","status":"broken","statusDetails":"selenium.common.exceptions.ElementNotInteractableException: Message: element not interactable\n  (Session info: chrome=123.0.6312.123)\nStacktrace:\n0   chromedriver                        0x000000010db9c0c8 chromedriver + 4595912\n1   chromedriver                        0x000000010db93e33 chromedriver + 4562483\n2   chromedriver                        0x000000010d797225 chromedriver + 381477\n3   chromedriver                        0x000000010d7e21ed chromedriver + 688621\n4   chromedriver                        0x000000010d7d590c chromedriver + 637196\n5   chromedriver                        0x000000010d803142 chromedriver + 823618\n6   chromedriver                        0x000000010d7d52b8 chromedriver + 635576\n7   chromedriver                        0x000000010d8032fe chromedriver + 824062\n8   chromedriver                        0x000000010d82314d chromedriver + 954701\n9   chromedriver                        0x000000010d802ee3 chromedriver + 823011\n10  chromedriver                        0x000000010d7d3be4 chromedriver + 629732\n11  chromedriver                        0x000000010d7d479e chromedriver + 632734\n12  chromedriver                        0x000000010db61fe2 chromedriver + 4358114\n13  chromedriver                        0x000000010db66c2d chromedriver + 4377645\n14  chromedriver                        0x000000010db665a3 chromedriver + 4375971\n15  chromedriver                        0x000000010db66ed5 chromedriver + 4378325\n16  chromedriver                        0x000000010db4ba05 chromedriver + 4266501\n17  chromedriver                        0x000000010db6725d chromedriver + 4379229\n18  chromedriver                        0x000000010db3e050 chromedriver + 4210768\n19  chromedriver                        0x000000010db84a98 chromedriver + 4500120\n20  chromedriver                        0x000000010db84c11 chromedriver + 4500497\n21  chromedriver                        0x000000010db93a73 chromedriver + 4561523\n22  libsystem_pthread.dylib             0x00007ff816bfb18b _pthread_start + 99\n23  libsystem_pthread.dylib             0x00007ff816bf6ae3 thread_start + 15","time":{"start":1713139083114,"stop":1713139170916,"duration":87802}}]},"cb965b49d65ea300604de77642d2ac9b":{"statistic":{"failed":0,"broken":1,"skipped":0,"passed":0,"unknown":0,"total":1},"items":[{"uid":"44d47231762c63ca","status":"broken","statusDetails":"selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {\"method\":\"xpath\",\"selector\":\"//p[@class='font-lightgrey bold'][normalize-space()='2']\"}\n  (Session info: chrome=123.0.6312.123); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception\nStacktrace:\n0   chromedriver                        0x000000010eae70c8 chromedriver + 4595912\n1   chromedriver                        0x000000010eadee33 chromedriver + 4562483\n2   chromedriver                        0x000000010e6e239a chromedriver + 381850\n3   chromedriver                        0x000000010e72bf08 chromedriver + 683784\n4   chromedriver                        0x000000010e72c191 chromedriver + 684433\n5   chromedriver                        0x000000010e770b14 chromedriver + 965396\n6   chromedriver                        0x000000010e74e16d chromedriver + 823661\n7   chromedriver                        0x000000010e76e14d chromedriver + 954701\n8   chromedriver                        0x000000010e74dee3 chromedriver + 823011\n9   chromedriver                        0x000000010e71ebe4 chromedriver + 629732\n10  chromedriver                        0x000000010e71f79e chromedriver + 632734\n11  chromedriver                        0x000000010eaacfe2 chromedriver + 4358114\n12  chromedriver                        0x000000010eab1c2d chromedriver + 4377645\n13  chromedriver                        0x000000010eab15a3 chromedriver + 4375971\n14  chromedriver                        0x000000010eab1ed5 chromedriver + 4378325\n15  chromedriver                        0x000000010ea96a05 chromedriver + 4266501\n16  chromedriver                        0x000000010eab225d chromedriver + 4379229\n17  chromedriver                        0x000000010ea89050 chromedriver + 4210768\n18  chromedriver                        0x000000010eacfa98 chromedriver + 4500120\n19  chromedriver                        0x000000010eacfc11 chromedriver + 4500497\n20  chromedriver                        0x000000010eadea73 chromedriver + 4561523\n21  libsystem_pthread.dylib             0x00007ff816bfb18b _pthread_start + 99\n22  libsystem_pthread.dylib             0x00007ff816bf6ae3 thread_start + 15","time":{"start":1713139200652,"stop":1713139219021,"duration":18369}}]},"2f1d58db96134f6ceb319001d1ae0c59":{"statistic":{"failed":1,"broken":0,"skipped":0,"passed":0,"unknown":0,"total":1},"items":[{"uid":"fd7b38fa0dca3bf3","status":"failed","statusDetails":"AssertionError: ++++ soft assert failure details follow below ++++\n\n++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++\nThe following 6 failures were found in \"wrapper\" (utilities.utils.Utils):\n+----------------------+----------------------+----------------------+\n Failure 1 (\"wrapper\" method)\n+----------------------+----------------------+----------------------+\nTraceback (most recent call last):\n  File \"/Users/k-bhatt/PythonTestFrameworkDemo/utilities/utils.py\", line 11, in assertListItemText\n    self.soft_assert(self.assertEqual, stop.text, value)\n  File \"/Users/k-bhatt/PythonTestFrameworkDemo/testcases/test_searchflights.py\", line 39, in test_search_flights_1_stop\n    self.utilPage.assertListItemText(allstops, stops)\n  File \"/Users/k-bhatt/PythonTestFrameworkDemo/env/lib/python3.11/site-packages/ddt.py\", line 221, in wrapper\n    return func(self, *args, **kwargs)\n  File \"/Users/k-bhatt/PythonTestFrameworkDemo/env/lib/python3.11/site-packages/softest/case.py\", line 62, in soft_assert\n    assert_method(*arguments, **keywords)\n  File \"/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/unittest/case.py\", line 873, in assertEqual\n    assertion_func(first, second, msg=msg)\n  File \"/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/unittest/case.py\", line 1253, in assertMultiLineEqual\n    self.fail(self._formatMessage(msg, standardMsg))\n  File \"/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/unittest/case.py\", line 703, in fail\n    raise self.failureException(msg)\nAssertionError: '2 Stops' != '2 Stop'\n- 2 Stops\n?       -\n+ 2 Stop\n\n\n-+ [1/6] +-\n\n+----------------------+----------------------+----------------------+\n Failure 2 (\"wrapper\" method)\n+----------------------+----------------------+----------------------+\nTraceback (most recent call last):\n  File \"/Users/k-bhatt/PythonTestFrameworkDemo/utilities/utils.py\", line 11, in assertListItemText\n    self.soft_assert(self.assertEqual, stop.text, value)\n  File \"/Users/k-bhatt/PythonTestFrameworkDemo/testcases/test_searchflights.py\", line 39, in test_search_flights_1_stop\n    self.utilPage.assertListItemText(allstops, stops)\n  File \"/Users/k-bhatt/PythonTestFrameworkDemo/env/lib/python3.11/site-packages/ddt.py\", line 221, in wrapper\n    return func(self, *args, **kwargs)\n  File \"/Users/k-bhatt/PythonTestFrameworkDemo/env/lib/python3.11/site-packages/softest/case.py\", line 62, in soft_assert\n    assert_method(*arguments, **keywords)\n  File \"/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/unittest/case.py\", line 873, in assertEqual\n    assertion_func(first, second, msg=msg)\n  File \"/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/unittest/case.py\", line 1253, in assertMultiLineEqual\n    self.fail(self._formatMessage(msg, standardMsg))\n  File \"/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/unittest/case.py\", line 703, in fail\n    raise self.failureException(msg)\nAssertionError: '2 Stops' != '2 Stop'\n- 2 Stops\n?       -\n+ 2 Stop\n\n\n-+ [2/6] +-\n\n+----------------------+----------------------+----------------------+\n Failure 3 (\"wrapper\" method)\n+----------------------+----------------------+----------------------+\nTraceback (most recent call last):\n  File \"/Users/k-bhatt/PythonTestFrameworkDemo/utilities/utils.py\", line 11, in assertListItemText\n    self.soft_assert(self.assertEqual, stop.text, value)\n  File \"/Users/k-bhatt/PythonTestFrameworkDemo/testcases/test_searchflights.py\", line 39, in test_search_flights_1_stop\n    self.utilPage.assertListItemText(allstops, stops)\n  File \"/Users/k-bhatt/PythonTestFrameworkDemo/env/lib/python3.11/site-packages/ddt.py\", line 221, in wrapper\n    return func(self, *args, **kwargs)\n  File \"/Users/k-bhatt/PythonTestFrameworkDemo/env/lib/python3.11/site-packages/softest/case.py\", line 62, in soft_assert\n    assert_method(*arguments, **keywords)\n  File \"/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/unittest/case.py\", line 873, in assertEqual\n    assertion_func(first, second, msg=msg)\n  File \"/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/unittest/case.py\", line 1253, in assertMultiLineEqual\n    self.fail(self._formatMessage(msg, standardMsg))\n  File \"/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/unittest/case.py\", line 703, in fail\n    raise self.failureException(msg)\nAssertionError: '' != '2 Stop'\n+ 2 Stop\n\n-+ [3/6] +-\n\n+----------------------+----------------------+----------------------+\n Failure 4 (\"wrapper\" method)\n+----------------------+----------------------+----------------------+\nTraceback (most recent call last):\n  File \"/Users/k-bhatt/PythonTestFrameworkDemo/utilities/utils.py\", line 11, in assertListItemText\n    self.soft_assert(self.assertEqual, stop.text, value)\n  File \"/Users/k-bhatt/PythonTestFrameworkDemo/testcases/test_searchflights.py\", line 39, in test_search_flights_1_stop\n    self.utilPage.assertListItemText(allstops, stops)\n  File \"/Users/k-bhatt/PythonTestFrameworkDemo/env/lib/python3.11/site-packages/ddt.py\", line 221, in wrapper\n    return func(self, *args, **kwargs)\n  File \"/Users/k-bhatt/PythonTestFrameworkDemo/env/lib/python3.11/site-packages/softest/case.py\", line 62, in soft_assert\n    assert_method(*arguments, **keywords)\n  File \"/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/unittest/case.py\", line 873, in assertEqual\n    assertion_func(first, second, msg=msg)\n  File \"/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/unittest/case.py\", line 1253, in assertMultiLineEqual\n    self.fail(self._formatMessage(msg, standardMsg))\n  File \"/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/unittest/case.py\", line 703, in fail\n    raise self.failureException(msg)\nAssertionError: '2 Stops' != '2 Stop'\n- 2 Stops\n?       -\n+ 2 Stop\n\n\n-+ [4/6] +-\n\n+----------------------+----------------------+----------------------+\n Failure 5 (\"wrapper\" method)\n+----------------------+----------------------+----------------------+\nTraceback (most recent call last):\n  File \"/Users/k-bhatt/PythonTestFrameworkDemo/utilities/utils.py\", line 11, in assertListItemText\n    self.soft_assert(self.assertEqual, stop.text, value)\n  File \"/Users/k-bhatt/PythonTestFrameworkDemo/testcases/test_searchflights.py\", line 39, in test_search_flights_1_stop\n    self.utilPage.assertListItemText(allstops, stops)\n  File \"/Users/k-bhatt/PythonTestFrameworkDemo/env/lib/python3.11/site-packages/ddt.py\", line 221, in wrapper\n    return func(self, *args, **kwargs)\n  File \"/Users/k-bhatt/PythonTestFrameworkDemo/env/lib/python3.11/site-packages/softest/case.py\", line 62, in soft_assert\n    assert_method(*arguments, **keywords)\n  File \"/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/unittest/case.py\", line 873, in assertEqual\n    assertion_func(first, second, msg=msg)\n  File \"/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/unittest/case.py\", line 1253, in assertMultiLineEqual\n    self.fail(self._formatMessage(msg, standardMsg))\n  File \"/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/unittest/case.py\", line 703, in fail\n    raise self.failureException(msg)\nAssertionError: '2 Stops' != '2 Stop'\n- 2 Stops\n?       -\n+ 2 Stop\n\n\n-+ [5/6] +-\n\n+----------------------+----------------------+----------------------+\n Failure 6 (\"wrapper\" method)\n+----------------------+----------------------+----------------------+\nTraceback (most recent call last):\n  File \"/Users/k-bhatt/PythonTestFrameworkDemo/utilities/utils.py\", line 11, in assertListItemText\n    self.soft_assert(self.assertEqual, stop.text, value)\n  File \"/Users/k-bhatt/PythonTestFrameworkDemo/testcases/test_searchflights.py\", line 39, in test_search_flights_1_stop\n    self.utilPage.assertListItemText(allstops, stops)\n  File \"/Users/k-bhatt/PythonTestFrameworkDemo/env/lib/python3.11/site-packages/ddt.py\", line 221, in wrapper\n    return func(self, *args, **kwargs)\n  File \"/Users/k-bhatt/PythonTestFrameworkDemo/env/lib/python3.11/site-packages/softest/case.py\", line 62, in soft_assert\n    assert_method(*arguments, **keywords)\n  File \"/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/unittest/case.py\", line 873, in assertEqual\n    assertion_func(first, second, msg=msg)\n  File \"/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/unittest/case.py\", line 1253, in assertMultiLineEqual\n    self.fail(self._formatMessage(msg, standardMsg))\n  File \"/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/unittest/case.py\", line 703, in fail\n    raise self.failureException(msg)\nAssertionError: '' != '2 Stop'\n+ 2 Stop\n\n-+ [6/6] +-","time":{"start":1713139056517,"stop":1713139078021,"duration":21504}}]},"3f2fd62a2f2f88901b00cf4dc563d187":{"statistic":{"failed":0,"broken":1,"skipped":0,"passed":0,"unknown":0,"total":1},"items":[{"uid":"b9dd7c3bb9c846eb","status":"broken","statusDetails":"selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {\"method\":\"xpath\",\"selector\":\"//p[@class='font-lightgrey bold'][normalize-space()='1']\"}\n  (Session info: chrome=123.0.6312.123); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception\nStacktrace:\n0   chromedriver                        0x0000000108b010c8 chromedriver + 4595912\n1   chromedriver                        0x0000000108af8e33 chromedriver + 4562483\n2   chromedriver                        0x00000001086fc39a chromedriver + 381850\n3   chromedriver                        0x0000000108745f08 chromedriver + 683784\n4   chromedriver                        0x0000000108746191 chromedriver + 684433\n5   chromedriver                        0x000000010878ab14 chromedriver + 965396\n6   chromedriver                        0x000000010876816d chromedriver + 823661\n7   chromedriver                        0x000000010878814d chromedriver + 954701\n8   chromedriver                        0x0000000108767ee3 chromedriver + 823011\n9   chromedriver                        0x0000000108738be4 chromedriver + 629732\n10  chromedriver                        0x000000010873979e chromedriver + 632734\n11  chromedriver                        0x0000000108ac6fe2 chromedriver + 4358114\n12  chromedriver                        0x0000000108acbc2d chromedriver + 4377645\n13  chromedriver                        0x0000000108acb5a3 chromedriver + 4375971\n14  chromedriver                        0x0000000108acbed5 chromedriver + 4378325\n15  chromedriver                        0x0000000108ab0a05 chromedriver + 4266501\n16  chromedriver                        0x0000000108acc25d chromedriver + 4379229\n17  chromedriver                        0x0000000108aa3050 chromedriver + 4210768\n18  chromedriver                        0x0000000108ae9a98 chromedriver + 4500120\n19  chromedriver                        0x0000000108ae9c11 chromedriver + 4500497\n20  chromedriver                        0x0000000108af8a73 chromedriver + 4561523\n21  libsystem_pthread.dylib             0x00007ff816bfb18b _pthread_start + 99\n22  libsystem_pthread.dylib             0x00007ff816bf6ae3 thread_start + 15","time":{"start":1713139178424,"stop":1713139195285,"duration":16861}}]}}